app/src/main/java/openfoodfacts/github/scrachx/openfood/features/scan/ContinuousScanActivity.kt

```diff
@@ -223,7 +225,7 @@ class ContinuousScanActivity : BaseActivity(), IProductView {
                         .apply { setGravity(CENTER, 0, 0) }
                         .show()
 
-                    Log.w(LOG_TAG, err.message, err)
```

Comment:
Why?

Reply:
because LOG_TAG uses explicit unwrap here, I'm trying to avoid using `!!` everywhere just because it requires always think "is it safe to use !! here or not? does it crash on production?"
and we use this tag only in one place so I changed it to a simple string